### ISSUE TITLE
Qt errors on startup

### ISSUE THREAD

**Issue by Subv** [citra-emu/citra#449]
```
QObject::connect: Cannot connect (null)::screenChanged(QScreen*) to GRenderWindow::OnFramebufferSizeChanged()
QObject::connect: No such slot GraphicsFramebufferWidget::OnFramebufferWidthChanged(int) in ..\..\..\src\citra_qt\debugger\graphics_framebuffer.cpp:87
QObject::connect:  (receiver name: 'PicaFramebuffer')
QObject::connect: No such slot GraphicsFramebufferWidget::OnFramebufferHeightChanged(int) in ..\..\..\src\citra_qt\debugger\graphics_framebuffer.cpp:88 
QObject::connect:  (receiver name: 'PicaFramebuffer')
```
They've been happening for a while, I'm not sure when it started

**Comment by Subv** [comment]
@neobrain

**Comment by purpasmart96** [comment]
Same here.

**Comment by tgharib** [comment]
Someone fixed the other Qt errors except this one:

`QObject::connect: Cannot connect (null)::screenChanged(QScreen*) to GRenderWindow::OnFramebufferSizeChanged()`

It is caused by commit 40acd63631eeaca248b2393cde399714a730214f on Nov 18. It is caused by this line: `connect(this->windowHandle(), SIGNAL(screenChanged(QScreen*)), this, SLOT(OnFramebufferSizeChanged()));` in src/citra_qt/bootmanager.cpp. It seems that the issue is that this->windowHandle() is equal to 0.

**Comment by neobrain** [comment]
This was fixed in PR #671, but that branch was never finished.

**Comment by TheOddler** [comment]
I get this bug too. Seems like an old problem, was it never fixed?
